ngraph_bridge/pass/transpose_sinking.cc

```diff
@@ -105,11 +105,15 @@ static void write_transposemap(TransposeMap& reorders,
 
 static shared_ptr<opset::Transpose> read_transposemap(
     TransposeMap& reorders, ngraph::Output<ngraph::Node> target) {
+  // if(reorders.find(target.get_node_shared_ptr()) != reorders.end()) {
```

Comment:
Is this needed? Do we ever try to read transposes that are not in the transposemap?

Reply:
No, this was done while debugging the model and there was an issue with reading the map for which there wasn't an entry. I'll remove, don't think we'll hit this issue.